Mathlib/MeasureTheory/Integral/linearRMK.lean

```diff
@@ -51,25 +51,25 @@ lemma mono (f₁ f₂ : C_c(X, ℝ)) (h : f₁.1 ≤ f₂.1) : Λ f₁ ≤ Λ f
 
 end PositiveLinear
 
-section linearRMK
+section LinearRMK
 
 variable [T2Space X] [LocallyCompactSpace X] [MeasurableSpace X] [BorelSpace X]
 
 /-- The measure induced for `Real`-linear positive functional `Λ`, defined through `toNNRealLinear`
 and the `NNReal`-version of `rieszContent`. -/
-def μ := (rieszContent (toNNRealLinear hΛ)).measure
+def rieszMeasure := (rieszContent (toNNRealLinear hΛ)).measure
 
 /-- If `f` assumes values between `0` and `1` and the support is contained in `K`, then
-`Λ f ≤ μ K`. -/
+`Λ f ≤ rieszMeasure K`. -/
 lemma leRieszMeasure_of_Compacts {f : C_c(X, ℝ)} (hf : ∀ (x : X), 0 ≤ f x ∧ f x ≤ 1)
-    {K : Compacts X} (h : tsupport f ⊆ K) : ENNReal.ofReal (Λ f) ≤ (μ Λ hΛ) K := by
+    {K : Compacts X} (h : tsupport f ⊆ K) : ENNReal.ofReal (Λ f) ≤ (rieszMeasure Λ hΛ) K := by
```

Comment:
```suggestion
    {K : Compacts X} (h : tsupport f ⊆ K) : (Λ f).ofReal ≤ rieszMeasure Λ hΛ K := by
```

Reply:
here `ofReal` is correct, but the dot notation does not work